.github/workflows/pre-release-continuous-test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Run continuous pre release testing
+on:
+  schedule:
+    # Run hourly
+    - cron: "0 * * * *"
+
+permissions:
+  contents: read
+  id-token: write #required for GCP Workload Identity federation which we use to login into Google Artifact Registry
+
+jobs:
+  run-forge:
+    uses: ./.github/workflows/run-forge.yaml
+    with:
+      GIT_SHA: HEAD
```

Comment:
It's implied that GIT_SHA == IMAGE_TAG in our docker image build workflow. We probably need to just get the sha of the HEAD of main, e.g. with `github.sha`